### DISCUSSION TITLE
Strapi emails not working well in 19.5 - 20.1

### DISCUSSION THREAD

**Discussion by lsliwaradioluz** [strapi/strapi#6041] [Legacy Topics]
Attempting to send an email via Strapi's Sendmail or Sendgrid returns 500: Internal Server Error. You can see me reproduce the bug with a brand new Strapi (v20.1) project below.

https://www.loom.com/share/d237e392ec2d41cfbaa931d884e611e1

In the video, I explore three cases: 
1. Attempting to send an email to any address with Sendgrid: error,
2. Attempting to send an email to hesacademyapp@gmail.com with Sendmail: success, 
3. Attempting to send an email to info@hesacademy.pl with Sendmail: error, 

I have to mention I used the Sendgrid provider in the past (release 17.5) and it worked perfect. Do you have any idea why this errors happen and why some emails cause them whereas some don't?

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions)

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Comment by lauriejim** [comment]
Hello @lsliwaradioluz 

I just tested on my side and it works well - https://www.loom.com/share/d9e963c85355444b8b04e3e9fe30da68

For the local setup, it's normal, it depends on your machine setup.
